optimize_textures.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Colors with 0 alpha need to be preserved, because opaque leaves ignore alpha.
+# For that purpose, the use of indexed colors is disabled (-nc).
```

Comment:
Please put this in an "utils" folder in case an .obj compression script is added in the future.

Reply:
This is fine by me

Reply:
Okay, will do. This brings up the question: should I go ahead and add the compress.obj script to the same utils folder in #2807? Then, should I add a readme.md to that folder saying how to use both scripts in a new PR?